jobs_worker.go

```diff
@@ -0,0 +1,133 @@
+package foundation
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/gocraft/work"
+	"github.com/gomodule/redigo/redis"
+)
+
+const (
+	defaultConcurrency = 5
+	defaultNamespace   = "foundation_jobs_worker"
```

Comment:
let's meybe share those with the ones in `jobs/enqueuer`? At least, the `namespace`.

Reply:
Add import from jobs/enqueuer